### DISCUSSION TITLE
Regex search is case insensitive

### DISCUSSION THREAD

**Discussion by tejasvi** [community/community#10708] [Other Feature Feedback, Questions, & Ideas] [answered]
Example: https://cs.github.com/?q=%2Fatool.*-X%2F matches `atool -x` instead of `atool -X`.
The case-insensitive nature of regexes invalidates their entire purpose. It would be great if CS is made case sensitive.

**Comment by look** [comment, marked as answer]
Case sensitive search is on our roadmap. See https://github.com/github/feedback/discussions/9759#discussioncomment-1929045

**Comment by maxkosty** [comment]
This can be done with `(?-i)`:

e.g. `/(?-i)atool -X/`
